frontend/src/services/set-validation-language.ts

```diff
@@ -0,0 +1,19 @@
+
+import {Language, setLanguage} from "zero-zummon"
+
+export function setValidationLanguage(): void {
+    for (const language of navigator.languages) {
+        if (language.startsWith(Language.en.name)) {
+            setLanguage(Language.en)
+            return
+        }
+
+        if (language.startsWith(Language.nl.name)) {
+            setLanguage(Language.nl)
+            return
+        }
+    }
+
+    // No matching language.
```

Comment:
Reduce comments, cleaner to see

Reply:
I want to communicate to the reader that this case has been thought about.